src/Database/Schema/PostgresSchema.php

```diff
@@ -340,6 +345,7 @@ public function columnSql(Table $table, $name)
             'datetime' => ' TIMESTAMP',
             'timestamp' => ' TIMESTAMP',
             'uuid' => ' UUID',
+            'json' => ' JSON'
```

Comment:
9.2 was released 5 years ago, and probably people are not creating new software on top of it.

I also have the impression that postgres people are more eager to upgrade often than mysql users.